core/bin/oauth.rs

```diff
@@ -43,13 +43,30 @@ async fn index() -> &'static str {
 struct ConnectionCreatePayload {
     provider: ConnectionProvider,
     metadata: serde_json::Value,
+    // Optionally present secret fields (migration case).
```

Comment:
nit: wondering if it would be a bit cleaner to regroup these fields in a `migrated_connection_details` field or something like that

Reply:
sure happy to